### PR TITLE
[stable/grafana] Add namespace for grafana sidecar configmap scan.

### PR DESCRIPTION
**Pull request by arturo-c**
In order to be able to run against specific namespace or `ALL` https://github.com/kiwigrid/k8s-sidecar#configuration-environment-variables

### CODE REVIEW

**Review by giacope — CHANGES_REQUESTED** [review]
This looks good and I'm actually surprised that a PR was made here that quickly since my changes to the k8s-sidecar got merged the same day!

However, can you please add the same configuration to the data-sources sidecar?

**Review by stealthybox — CHANGES_REQUESTED** [review]
It looks like this changes the default behavior of the chart to useCurrentNamespace instead of `"ALL"` namespaces.

This is a backwards incompatible change on a stable chart.
We'll probably need to change the default to `"ALL"`.
_____________________
It would also be good that when the sidecar is configured to use the current namespace, that the existing `ClusterRole` in the chart is downgraded to a `Role` as demonstrated in https://github.com/helm/charts/pull/7060

See comment for justification: https://github.com/helm/charts/pull/7060#issuecomment-422497345

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.14.6
+version: 1.14.7
```
Comment (davidkarlsen) [inline stable/grafana/Chart.yaml:2]: new feature - 1.15.0

Comment (davidkarlsen) [reply stable/grafana/Chart.yaml:2]: also see @stealthybox comments

Comment (stealthybox) [reply stable/grafana/Chart.yaml:2]: this will need to be bumped higher -- I think 1.17.0 or 1.18.0

**Pull request closed without merging**